javaagent-bootstrap/src/main/java/io/opentelemetry/javaagent/bootstrap/ClassFileTransformerHolder.java

```diff
@@ -16,11 +16,21 @@ public final class ClassFileTransformerHolder {
 
   private static volatile ClassFileTransformer classFileTransformer;
 
-  public static ClassFileTransformer getClassFileTransformer() {
+  /**
+   * get class transformer for defining lambdas
+   *
+   * @return class transformer for defining lambdas
+   */
+  public static ClassFileTransformer getLambdaClassFileTransformer() {
```

Comment:
[for reviewer] this is only used for lambda, and the `ClassFileTransformer` instances that are used here are very specific to lambda instrumentation, so renaming helps keep it clear.